dll/win32/newdev/lang/ro-RO.rc

```diff
@@ -15,12 +15,12 @@ CAPTION "Expert de instalare dispozitiv"
 FONT 8, "MS Shell Dlg"
 BEGIN
     LTEXT "Bun venit!", IDC_WELCOMETITLE, 120, 8, 195, 16
-    LTEXT "Acest expert va instala un nou driver de dispozitiv pentru:", IDC_STATIC, 120, 21, 195, 16
-    LTEXT "Apăsați Înainte pentru a continua.", IDC_STATIC, 120, 169, 195, 16
+    LTEXT "Expertul facilitează instalarea de produse software pentru:", IDC_STATIC, 120, 21, 195, 16
```

Comment:
Why "produse software"? This interprets as any software that could be installed, a game is also a "produs". The previous translation was fine.

Reply:
You're right. I'll revert it.

Reply:
Fixed!